packages/@webex/plugin-meetings/src/meetings/index.ts

```diff
@@ -1064,9 +1065,15 @@ export default class Meetings extends WebexPlugin {
         .fetchInfoOptions(destination, type)
         // Catch a failure to fetch info options.
         .catch((error) => {
-          LoggerProxy.logger.info(
-            `Meetings:index#create --> INFO, unable to determine info options: ${error.message}`
+          LoggerProxy.logger.error(
+            `Meetings:index#create --> ERROR, unable to determine info options: ${error.message}`
           );
+          if (error.error === 'SPACE_ID_Depreciated') {
```

Comment:
lets try to add error codes so that we can later add them to developer docs

Reply:
check other SDKs like agora or twilio on how they do the error mapping

Reply:
Sure, @arun3528, Thanks..

Reply:
@arun3528, Error code 101 was added as Space ID deprecated as of beta, Please refer migration guide. Took reference from https://docs.agora.io/en/3.x/voice-calling/reference/error-codes